docs/new-cosing.md

```diff
@@ -0,0 +1,13 @@
+## Costing Rules
```

Comment:
Should be moved to a separate PR on top of this branch. Such unfinished documents usually stay unfinished for long time with no proper control.

Reply:
removed